keras/backend/common.py

```diff
@@ -108,6 +108,29 @@ def cast_to_floatx(x):
     return np.asarray(x, dtype=_FLOATX)
 
 
+def is_placeholder(tensor):
+    """Returns whether a tensor is a placeholder.
+
+    # Arguments
+        tensor: A tensor instance.
+
+    # Returns
+        A boolean.
+
+    # Example
+    ```python
+        >>> from keras import backend as K
+        >>> a = K.placeholder((2, 2), sparse=False)
+        >>> print(K.is_placeholder(a))
+        True
+    ```
+    """
+    try:
+        return tensor._is_placeholder
+    except AttributeError:
+        return True
+
```

Comment:
Yes, a change is still needed. What's a name that clearly specifies "Is there a need to supply data from python for this op"?